lightningd/notification.c

```diff
@@ -638,3 +638,45 @@ void notify_log(struct lightningd *ld, const struct log_entry *l)
 	log_notification_serialize(n->stream, l);
 	notify_send(ld, n);
 }
+
+static void plugin_started_notification_serialize(struct json_stream *stream,
+						  struct plugin *plugin)
+{
+	json_add_string(stream, "plugin_name", plugin->shortname);
+	json_add_string(stream, "plugin_path", plugin->cmd);
+	json_array_start(stream, "methods");
+	for (size_t i = 0; i < tal_count(plugin->methods); i++) {
+		json_add_string(stream, NULL, plugin->methods[i]);
+	}
+	json_array_end(stream);
+}
```

Comment:
if this is the same as `plugin_stopped_notificatoin_serialize` i think you can reasonably have one `plugin_notifiaction_serialize` method and use it for both?

Reply:
Yep, good point. Made that change